### PR TITLE
Add all RTTCU player cards

### PR DESCRIPTION
**Pull request by zzorba**
Leaked cards, release date seems fuzzy but seems like at least Canada is getting it next week.

### CODE REVIEW

**Comment by zzorba** [comment]
@drawntotheflame if you have a chance to review, would love to get this released to unlock releasing the RTTCU campaign.

File: pack/return/rttcu.json [hunk 1]
```diff
@@ -1,52 +1,254 @@
...
+        "deck_limit": 1,
+        "faction_code": "guardian",
+        "illustrator": "John Pacer",
+        "name": "Hallowed Mirror",
+        "pack_code": "rttcu",
+        "position": 2,
+        "quantity": 1,
+        "slot": "Accessory",
+        "skill_willpower": 2,
+        "text": "Limit 1 per deck.\n[reaction] When you play a copy of Soothing Melody: Either change each \"2\" on it to a \"3,\" or shuffle it into your deck instead of discarding it.\n</b> - After Hallowed Mirror enter play: Search your bonded cards for 3 copies of Soothing Melody. Add 1 to your hand and shuffle the other 2 into your deck.",
```
Comment (drawntotheflame) [inline pack/return/rttcu.json:31]: for the third line here, you've lost the opening bold tag and "Forced".

File: pack/return/rttcu.json [hunk 2]
```diff
@@ -1,52 +1,254 @@
...
+        "deck_limit": 2,
+        "faction_code": "rogue",
+        "illustrator": "Robert Laskey",
+        "name": "Knight of Swords",
+        "pack_code": "rttcu",
+        "position": 5,
+        "quantity": 2,
+        "slot": "Tarot",
+        "subname": "Charge Ever Onward",
+        "text": "[reactionn] When you would succeed at a skill test: You get +1 skill value for this test. You may discard Knight of Swords to get +3 instead.\n[reaction] When the game begins, if Knight of Swords is in your opening hand: Put it into play.",
```
Comment (drawntotheflame) [inline pack/return/rttcu.json:82]: Typo: reaction has two Ns at the end.

File: pack/return/rttcu.json [hunk 3]
```diff
@@ -1,52 +1,254 @@
...
+        "slot": "Tarot",
+        "subname": "Every Trial a Lesson",
+        "text": "[reaction] When you draw a non-weakness encounter card, exhaust Nine of Rods: Cancel all of that card's effects and shuffle it into the encounter deck. Then, draw another card from the top of the encounter deck.\n[reaction] When the game begins, if Nine of Rods is in your opening hand: Put it into play.",
+        "traits": "Tarot.",
+        "type_code": "asset",
+        "xp": 3
+    },
+    {
+        "code": "54010",
+        "cost": 3,
```
Comment (drawntotheflame) [inline pack/return/rttcu.json:158]: Cost is 2.

File: pack/return/rttcu.json [hunk 4]
```diff
@@ -1,52 +1,254 @@
...
+        "type_code": "asset",
+        "xp": 2
+    },
+    {
+        "code": "54013",
+        "cost": null,
+        "deck_limit": 1,
+        "faction_code": "neutral",
+        "illustrator": "Allan Bednar",
+        "name": "Moon Pendant",
```
Comment (drawntotheflame) [inline pack/return/rttcu.json:212]: Name is 'Observed'.

File: pack/return/rttcu.json [hunk 5]
```diff
@@ -1,52 +1,254 @@
...
+    {
+        "code": "54015",
+        "cost": 3,
+        "deck_limit": 1,
+        "faction_code": "neutral",
+        "illustrator": "Robert Laskey",
+        "name": "The Devil • XV",
+        "pack_code": "rttcu",
+        "position": 15,
+        "quantity": 1,
```
Comment (drawntotheflame) [inline pack/return/rttcu.json:246]: Quantity is 2.

File: pack/return/rttcu.json [hunk 6]
```diff
@@ -1,52 +1,254 @@
...
+        "subtype_code": "basicweakness",
+        "text": "Permanent.\nBefore setting up each game, draw a random card from the tarot deck and place the chosen card in front of you, in the reverse position. Each other investigator ignores its effects.",
+        "traits": "Curse. Omen.",
+        "type_code": "treachery"
+    },
+    {
+        "code": "54015",
+        "cost": 3,
+        "deck_limit": 1,
```
Comment (drawntotheflame) [inline pack/return/rttcu.json:240]: Limit is 2 I think, because quantity is 2.

File: pack/return/rttcu_encounter.json [hunk 7]
```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "code": "54041",
+        "cost": 2,
+        "deck_limit": 1,
+        "encounter_code": "return_to_the_wages_of_sin",
+        "encounter_position": 9,
+        "faction_code": "neutral",
+        "health": 2,
+        "illustrator": "Dravenzka Kimpel",
```
Comment (drawntotheflame) [inline pack/return/rttcu_encounter.json:10]: 'Drazenka'

**Review by drawntotheflame — COMMENTED** [review]
Great job @zzorba , just a few little things.

File: pack/return/rttcu.json (hunk 2 above)
Comment (zzorba) [reply pack/return/rttcu.json:82]: thanks, my keyboard is breaking.  #apple problems.

File: pack/return/rttcu.json (hunk 6 above)
Comment (zzorba) [reply pack/return/rttcu.json:240]: Good to know, that's unfortunate cause this card looks mean.

**Comment by zzorba** [comment]
Updated, thank you for the review.  @Kamalisk I think this is good to go live now, though unclear what we should set release date for the pack to.

**Pull request merged**